jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -512,12 +525,30 @@ String getFormat() {
    */
   List<Path> getExtraDirectories() {
     // TODO: Should inform user about nonexistent directory if using custom directory.
-    String property = getProperty(PropertyNames.EXTRA_DIRECTORY_PATH);
+    String deprecatedProperty = getProperty(PropertyNames.EXTRA_DIRECTORY_PATH);
+    String newProperty = getProperty(PropertyNames.EXTRA_DIRECTORIES_PATHS);
+
+    List<File> deprecatedPaths = extraDirectory.getPaths();
+    List<File> newPaths = extraDirectories.getPaths();
+
+    if (deprecatedProperty != null) {
+      getLog()
+          .warn(
+              "The property 'jib.extraDirectory.path' is deprecated; "
```

Comment:
I think this error message can look a little more maven-y?
```c
"The configuration <jib><extraDirectory><path> is deprecated" // or something?
```

Reply:
This is specifically about this property though.

Reply:
Yeah, the usage is `-Djib.extraDirectory.path=...` or `<jib.extraDirectory.path>`. So they will understand.